cmd/construct.go

```diff
@@ -0,0 +1,129 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+)
+
+/* WARNING - THIS WILL WRITE THE README IN THE DIRECTORY RUNNED
+ * PLEASE HAVE CAUTION
+ */
+
+// Structure of the README
+
+type PromptStructure struct {
+	ErrorMsg string
+	Label    string
+}
+
+var generation = &cobra.Command{
+	Use:   "readme_generate",
+	Short: "Will generate and write your README",
+	Long:  "Will generate and write your README",
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Printf("WARNING - THIS WILL WRITE THE README IN THE DIRECTORY RUNNED PLEASE HAVE CAUTION\n")
+		fmt.Printf("\n")
+		fmt.Printf("All options are optional, if a option is not provided the value will be 'PlaceHolder'\n")
+		fmt.Printf("\n")
+		GenerateReadme()
+	},
+}
+
+func GetUserInput(pc PromptStructure) string {
```

Comment:
There is a util function called TakeInput that is doing this, probably got added after you made your own. sorry 😓, but please use that.

Reply:
Also assuming you already done this since my function got removed.